src/main/java/bookstore/controller/BookController.java

```diff
@@ -0,0 +1,36 @@
+package bookstore.controller;
+
+import bookstore.dto.BookDto;
+import bookstore.dto.CreateBookRequestDto;
+import bookstore.service.BookService;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping(value = "/books")
+public class BookController {
+
```

Comment:
About this empty line you can use this code style if you prefer but please make it consistent for each class;)